doc/memos/rdm-draft-chrysn-boardnames.md

```diff
@@ -0,0 +1,172 @@
+- RDM: draft-chrysn-boardnames
+- Title: Board names for RIOT boards
+- Author: Christian Amsüss
+- Status: Draft
+- Created: 2022-09
+
+## Abstract
+
+This memo describes how boards are assigned names in RIOT.
+
+## Status
+
+This document is a draft and reflects the author's opinion.
+
+## Terminology
+
+This document does not attempt to define what "board" means within RIOT;
+for the purpose of this document,
+a "board" is the name for the group of devices built with the same `BOARD` variable.
+A board's "technical name" is the corresponding value of the `BOARD` variable;
+its "friendly name" is the text by which it is shown in documentation.
+
+A board's "defining representative" is a distinct piece of hardware
+which the maintainers of that board picked.
+This distinction is only relevant for boards containing devices created by multiple vendors;
+typically, the other boards are (unauthorized) clones
+or forks (of Open Hardware boards)
+of the defining representative.
+
+# 0. Motivation
+
+**This section is to be removed by the RDM editor before publication.**
+
+As of now,
+we have boards listed under grossly misleading names
+([nRF52 DK](https://doc.riot-os.org/group__boards__nrf52dk.html)
+is documented to be "some nameless simple development boards"
+when there is [a vendor board of that same name](https://www.nordicsemi.com/Products/Development-hardware/nRF52-DK)),
+boards with no further information whatsoever
+(ducking for [avsextrem](https://doc.riot-os.org/group__boards__avsextrem.html)
+only finds the RIOT documentation, which is empty on that board)
+and boards with no clear defining representative
+(e.g. the [blackpill boards](https://doc.riot-os.org/group__boards__blackpill.html)).
+
+This is a mess.
+
+
```

Comment:
Don't forget ;-)

```suggestion
```

Reply:
Yeah, I think the time for motivating this has passed.